cfme/tests/infrastructure/test_tenant_quota.py

```diff
@@ -582,3 +583,133 @@ def test_quota_not_fails_after_vm_reconfigure_disk_remove(request, appliance, fu
+        tenant=f"My Company/{new_project.parent_tenant.name}/{new_project.name}",
+    )
+    yield group
+    group.delete_if_exists()
+
+
+@pytest.fixture(scope="module")
+def new_user_project(appliance, new_group_project):
+    """This fixture creates new user which is assigned to new group and project"""
+    user = appliance.collections.users.create(
+        name=fauxfactory.gen_alphanumeric(start="user_").lower(),
+        credential=Credential(principal=fauxfactory.gen_alphanumeric(start="uid"),
+                              secret=fauxfactory.gen_alphanumeric(start="pwd")),
+        email=fauxfactory.gen_email(),
+        groups=new_group_project,
+        cost_center="Workload",
+        value_assign="Database",
+    )
+    yield user
+    user.delete_if_exists()
+
+
+@pytest.mark.tier(2)
+@pytest.mark.meta(automates=[1456819, 1401251])
+@pytest.mark.parametrize(
+    ['set_project_quota', 'custom_prov_data', 'set_default'],
+    [
+        [(('cpu', '2'), ('storage', '0.01'), ('memory', '2')),
+         ({'hardware': {'num_sockets': '8', 'memory': '4096'}}), False]
+    ],
+    indirect=['set_project_quota', 'custom_prov_data', 'set_default'],
+    ids=['max_cpu_storage_memory']
+)
+@pytest.mark.parametrize('context', [ViaSSUI, ViaUI])
+@pytest.mark.provider([RHEVMProvider], scope='function')
+def test_simultaneous_tenant_quota(request, appliance, context, new_project, new_user_project,
+                                   set_project_quota, custom_prov_data, catalog_item, set_default):
+    """
+    Bugzilla:
+        1456819
+        1401251
+
+    Polarion:
+        assignee: ghubale
+        initialEstimate: 1/6h
+        caseimportance: medium
+        caseposneg: positive
+        testtype: functional
+        startsin: 5.8
+        casecomponent: Provisioning
+        tags: quota
+        setup:
+            1. Create tenant1 and tenant2.
+            2. Create a project under tenant1 or tenant2
+            3. Enable quota for cpu, memory or storage etc
+            4. Create a group and add role super_administrator
+            5. Create a user and add it to the group
+        testSteps:
+            1. Login with the newly created user in the service portal. Take multiple items which go
+               over the allocated quota
+        expectedResults:
+            1. CFME should deny request with quota exceeded reason
+    """
+    with new_user_project:
+        with appliance.context.use(context):
+            appliance.server.login(new_user_project)
+            service_catalogs = ServiceCatalogs(appliance, catalog_item.catalog, catalog_item.name)
+            if context is ViaSSUI:
+                service_catalogs.add_to_shopping_cart()
+            service_catalogs.order()
+    # nav to requests page to check quota validation
+    request_description = f'Provisioning Service [{catalog_item.name}] from [{catalog_item.name}]'
+    provision_request = appliance.collections.requests.instantiate(request_description)
```

Comment:
I think no need of instantiating provision request. Its already instantiate by `order` method.